### PR TITLE
Moved ClientSystem & ServerSystem Initialization 

### PR DESCRIPTION
**Pull request by norda97**
## Purpose
  - What value is this change providing?
Moved initialization of ClientSystem & ServerSystem before initialization of the rendergraph.
Furthermore disabled initialization LightRenderer, ParticleManager, ParticleRenderer and ParticleUpdater on the server side.

  - Why is this change being made or feature being added?
Server should not render particles nor lights also if the server initilized ParticleManager a crash was caused due to ASBuilder not being initilized on server

### CODE REVIEW

File: LambdaEngine/Source/Rendering/ParticleManager.cpp [hunk 1]
```diff
@@ -17,65 +17,69 @@ namespace LambdaEngine
 {
 	void ParticleManager::Init(uint32 maxParticleCapacity, ASBuilder* pASBuilder)
 	{
+		if (!m_Initilized)
+		{
+			m_Initilized = true;
```
Comment (frozenbanana) [inline LambdaEngine/Source/Rendering/ParticleManager.cpp]: spelling: ~m_Initilized~ -> m_Initialized

I would also set `m_Initialized` to true after you have run all necessary steps not before.

**Review by frozenbanana — CHANGES_REQUESTED** [review]
looks good. One small correction.

File: LambdaEngine/Source/Rendering/ParticleManager.cpp (hunk 1 above)
Comment (norda97) [reply LambdaEngine/Source/Rendering/ParticleManager.cpp]: Good spot!

**Review by frozenbanana — APPROVED** [review]
nice

**Pull request merged**